src/main/java/nl/tudelft/jpacman/npc/ghost/blinkyPursuit.java

```diff
@@ -0,0 +1,7 @@
+package nl.tudelft.jpacman.npc.ghost;
+
+/**
+ * Created by UMONS on 18/04/2016.
+ */
+public class blinkyPursuit {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class names should begin with an uppercase character](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2525539288/issues/source?bid=3256381&fileBranchId=3281825#l6)